docs/use/convert.md

```diff
@@ -11,17 +11,32 @@ The `spk convert` command can be used to ingest packages from supported package
 Pip packages from pypi can be converted into spk packages as well. This process will recursively find and convert any dependencies of the requested pip package as well.
 
 ```sh
-# convert the current version of filesequence
+# convert the current version of gitlab-python
 $ spk convert pip gitlab-python
 # or request a specific version (using pip version semantics)
 $ spk convert pip gitlab-python==1.7
 
-$ spk env gitlab-python --local -- python -c "import gitlab; print(gitlab)"
+# the imported packages will have python- prefixed to their name
+$ spk env python-gitlab-python --local -- python -c "import gitlab; print(gitlab)"
 ```
 
-As of writing, the converted package will also be dependant on the current os and arch since noarch support cannot easily be detected.
+> [!NOTE]
+> The `spk convert pip` command relies on an spk package called `spk-convert-pip`. We provide a recipe for this package in our github repo, which can be used to build and publish it for use in new environments. See [bootstrapping]({{< ref "../../admin/bootstrap" >}}).
+
+The converted package will also be dependant on the current os and arch since noarch support cannot easily be detected.
 
 ```sh
-# convert for a specific python version
-spk convert pip --python-version 2.7 numpy
+# convert for a specific python version, usually you want to override the abi as well
+spk convert pip --python-version 3.11 --python-abi=cp311m numpy
 ```
+
+## Other Package Sources
```

Comment:
this section is the only real new content, save a couple of tweaks here and there where I noticed old or incorrect versions and conventions (eg references to python 2.7, missing `python-` prefix callout for converted packages above...)